common/src/main/java/com/linecorp/centraldogma/internal/Jackson.java

```diff
@@ -77,7 +78,8 @@ public final class Jackson {
         compactMapper.enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
         prettyMapper.enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
 
-        registerModules(new SimpleModule().addSerializer(Instant.class, InstantSerializer.INSTANCE)
+        registerModules(new Jdk8Module(),
```

Comment:
Which one made you add this module?

Reply:
It was added to serialize `Optional` in `MirrorCredential` but I decided to change the return type. Let me remove it.
https://github.com/line/centraldogma/pull/880/files#diff-3fcccba8a8cdf811e08f7c99cf837dbcc5c6c5164ff32caeecf9db1d395a1ee0R68